contents/blog/startup-unified.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Startup Spotlight: Unified.to"
+date: 2023-06-19
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/spotlight_unified.png
+featuredImageType: full
+category: Startups
+---
+This month we’re chatting to [Unified.to](https://unified.to/) co-founder and CEO, Roy Pereira. Roy is a serial entrepreneur, and Unified.to is his fifth startup. Previously, Roy had been a GM at [Vendasta](/customers/vendasta), the world’s leading end-to-end platform for channel partners selling digital solutions to small and medium-sized businesses.
```

Comment:
Previously, Roy had been the CEO and founder of CalendarHero (aka. Zoom.ai), which was acquired in 2021 by Vendasta.

Reply:
```suggestion
This month we’re chatting to [Unified.to](https://unified.to/) co-founder and CEO, Roy Pereira. Roy is a serial entrepreneur, and Unified.to is his fifth startup. Previously, Roy had been the CEO and founder of CalendarHero (formerly known as Zoom.ai), which was acquired by [Vendasta](/customers/vendasta) in 2021.
```